modules/installation-special-config-crony.adoc

```diff
@@ -16,13 +16,14 @@ to your nodes as a MachineConfig.
 [source,terminal]
 ----
 $ cat << EOF | base64
-    server clock.redhat.com iburst
+    pool 0.rhel.pool.ntp.org iburst <1>
     driftfile /var/lib/chrony/drift
     makestep 1.0 3
     rtcsync
     logdir /var/log/chrony
     EOF
 ----
+<1> Specify any valid, reachable time source, or you can specify any of the following NTP servers: `1.rhel.pool.ntp.org`, `2.rhel.pool.ntp.org`, or `3.rhel.pool.ntp.org`.
```

Comment:
Would suggest breaking this into two sentences. Maybe:
Specify any valid, reachable time source. Alternatively, you can specify any of the following NTP servers: `1.rhel.pool.ntp.org`, `2.rhel.pool.ntp.org`, or `3.rhel.pool.ntp.org`.

Reply:
@jeana-redhat thanks!